.github/workflows/build.yml

```diff
@@ -78,7 +78,28 @@ jobs:
         docker exec emscripten emcc -v
         docker exec emscripten emcmake cmake .
         docker exec emscripten make -j 2 VERBOSE=1
-
+  build-arm:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [arm64]
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
```

Comment:
I guess github doesn't have support for arm64 runners yet?  Thats too bad.   Maybe once this is fixed we can move to real arm64 hardware: https://github.com/actions/runner-images/issues/5631

Reply:
No it doesn't support, in another issue doesn't look something they're keen to implement soon https://github.com/actions/runner-images/issues/2552 . That would be the best, with Docker is takes forever, will close this PR.